### DISCUSSION TITLE
CVE-2022-37734 on graphql-java

### DISCUSSION THREAD

**Discussion by bbakerman** [graphql-java/graphql-java#2958] [General]
Some people might start getting notified of a CVE for graphql-java which was released today.

https://www.cve.org/CVERecord?id=CVE-2022-37734

https://security.snyk.io/vuln/SNYK-JAVA-COMGRAPHQLJAVA-3021519

The Snynk document says

> How to fix?
> Upgrade com.graphql-java:graphql-java to version 19.0 or higher.

This is incomplete.  We backported the changes to older versions

The full list of fixed versions is

* v19.0
* v18.3
* v17.4

You don't have to upgrade to the latest graphql-java version to get this fix.

https://github.com/graphql-java/graphql-java/pull/2549

**Comment by dondonz** [comment]
We are trying to update the CVE text and we have informed Snyk, so that automated vulnerability detectors correctly register the patched versions. Note this may take a while.

In the meantime please do patch.

**Reply by dondonz** [reply]
Update: CVE text fixed! https://www.cve.org/CVERecord?id=CVE-2022-37734

**Reply by bbakerman** [reply]
https://security.snyk.io/vuln/SNYK-JAVA-COMGRAPHQLJAVA-3021519 - Snyk also updated
